chef/cookbooks/neutron/recipes/common_agent.rb

```diff
@@ -49,19 +49,20 @@
   subscribes :run, resources(:cookbook_file=> disable_rp_filter_file), :delayed
 end
 
+if neutron[:neutron][:networking_plugin] == "ml2"
+  ml2_mech_drivers = neutron[:neutron][:ml2_mechanism_drivers]
+  case
+  when ml2_mech_drivers.include?("openvswitch")
+    neutron_agent = node[:neutron][:platform][:ovs_agent_name]
+    neutron_agent_pkg = node[:neutron][:platform][:ovs_agent_pkg]
+    agent_config_path = "/etc/neutron/plugins/openvswitch/ovs_neutron_plugin.ini"
+  when ml2_mech_drivers.include?("linuxbridge")
+    neutron_agent = node[:neutron][:platform][:lb_agent_name]
+    neutron_agent_pkg = node[:neutron][:platform][:lb_agent_pkg]
+    agent_config_path = "/etc/neutron/plugins/linuxbridge/linuxbridge_conf.ini"
+  end
```

Comment:
One more question: I don't see any validation at https://github.com/crowbar/barclamp-neutron/pull/175/files#diff-1cab7f3b6842279ae4d6bd875e073cacR97 of the fact that ovs or linuxbridge must be specified. Is this not mandatory to have one of the two? If not, then where are we defining the variables above in that third case?